### PR TITLE
addpkg: bacon

### PR DESCRIPTION
**Pull request by FantasqueX**
According to https://riscv-notes.sh1mar.in/docs/record/rust-related#riscv64gc-unknown-linux-gnu

### CODE REVIEW

**Comment by Avimitin** [comment]
This can be simply done by removing the `--target` argument. Ref: https://github.com/felixonmars/archriscv-packages/pull/1025/commits/1a2691a1f5a42cbbe375cfaa15e6ae6f12b100ac

File: bacon/riscv64.patch [hunk 1]
```diff
@@ -0,0 +1,17 @@
+diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
```
Comment (cubercsl) [inline bacon/riscv64.patch:1]:
```suggestion
diff --git PKGBUILD PKGBUILD
```

Comment (Avimitin) [inline bacon/riscv64.patch:1]: We should remove the prefix `a/trunk` and `b/trunk` here.

**Comment by FantasqueX** [comment]
Fixed. Any other suggestions?

**Review by r-value — APPROVED** [review]
LGTM

**Pull request merged**